server/src/main/java/org/opensearch/index/compositeindex/datacube/startree/utils/ByteListBackedBitset.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.index.compositeindex.datacube.startree.utils;
+
+import org.apache.lucene.store.IndexOutput;
+import org.apache.lucene.store.RandomAccessInput;
+
+import java.io.IOException;
+
+/**
+ * This is used to store and retrieve null values in the bitset.
+ */
+public class ByteListBackedBitset {
```

Comment:
this does not look specific to star tree, should we place in a common place if someone wants to use it? wdyt?

Reply:
Yeah agreed. Will rename the file since we've moved from list to Array. Will move to `org.opensearch.common.util`